Changes.md

```diff
@@ -42,6 +42,12 @@ API
 - ScenePath : Added support for cancellation of calls to `isValid()` and `children()`.
 - FileSystemPath : Added support for cancellation of calls to `children()` and `property()`.
 - PathFilter : Added support for cancellation of calls to `filter()`.
+- Animation :
```

Comment:
We should also note that the crash when dragging keys around in the AnimationEditor has been fixed, in the Fixes section.